htmlmaker_preprocessing.ps1

```diff
@@ -32,35 +35,45 @@ Function LogWrite
    Add-content $Logfile -value "$logstring"
 }
 
-# if original file name is a .doc file
-$fileType = ".doc"
-If ($filenameSplit -eq $filename + $fileType)
```

Comment:
Removing test for .doc because we want to process all Word files--but should we replace with a test for any Word file type, and throw an error and quit Bookmaker if it's not a Word doc? Or are we already testing for that earlier? If we want to add that, reinstate the If block but change test to `($fileType -like ".doc?")`, and figure out how to handle sending an error message to the user.